ddtrace/contrib/trace_utils.py

```diff
@@ -58,18 +58,6 @@
 # Possible User Agent header.
 USER_AGENT_PATTERNS = ("http-user-agent", "user-agent")
 
-IP_PATTERNS = (
```

Comment:
This is a breaking change since `ddtrace/contrib/trace_utils.py` is technically in the public API. Can we keep the change here?